packages/core/src/modules/event/event-repository.ts

```diff
@@ -27,30 +57,78 @@ export class EventRepositoryImpl implements EventRepository {
       .where("id", "=", id)
       .returningAll()
       .executeTakeFirstOrThrow()
-    return mapToEvent(event)
+    return EventSchema.parse(event)
```

Comment:
create mapToEventWithCommittees, there should at least be a mapper function available so we don't write .parse() everywhere for consistency